### ISSUE TITLE
Augmentation

### ISSUE THREAD

**Issue by ansabsheikh9** [jeasinema/VoxelNet-tensorflow#46]
Hi @jeasinema,
I have noticed after augmantation of type 1, and adding delta Z makes some of the boxes are below ground plane. And some are above the ground plane. is it ok to give training data like this? 
could you please comment on this

![untitled](https://user-images.githubusercontent.com/32954628/42698297-c3e754a4-86bd-11e8-80d6-f560452e7afa.jpg)

**Comment by abhigoku10** [comment]
@jeasinema hi how did u augment the data can you pls share the steps . i too want ot augment the data but not able to do it

**Comment by ansabsheikh9** [comment]
Hi @abhigoku10 here [https://github.com/jeasinema/VoxelNet-tensorflow/blob/d24c3eefc5762e4785ca3be6ca7d1fd0ef16bc61/utils/data_aug.py#L23](url) is the function which is used for data augmentation.

**Comment by abhigoku10** [comment]
@ansabsheikh9  thanks for the pointers , have u done occlusion augmentation.Can you pls share some details on that

**Comment by ansabsheikh9** [comment]
@abhigoku10 I do not have any idea about occolusion augmentation, 
But here are three types of augmentations. For each iteration, one type of augmentation is performed which is selected based on some probability. First is random orientation of the whole scene, second is random magnification, third is augmenting individual object by rotation and translation in three axis.

**Comment by abhigoku10** [comment]
@ansabsheikh9  thanks for the information

**Comment by abhigoku10** [comment]
@ansabsheikh9  where you able to draw the bounding on the image??
